client/consensus/babe/src/lib.rs

```diff
@@ -119,9 +119,9 @@ use sp_api::ApiExt;
 
 mod aux_schema;
 mod verification;
-mod authorship;
+pub mod authorship;
 #[cfg(test)]
-mod tests;
+pub mod tests;
```

Comment:
```suggestion
mod tests;
```

It's not likely that this is necessary.